app/assets/javascripts/discourse/app/components/modal/dismiss-read.hbs

```diff
@@ -8,6 +8,7 @@
       <PreferenceCheckbox
         @labelKey="topics.bulk.also_dismiss_topics"
         @checked={{this.dismissTopics}}
+        class="stop-tracking"
```

Comment:
```suggestion
        class="dismiss-read-modal__stop-tracking"
```

Minor, but we should be following BEM rules when making CSS classes

Reply:
Nice one, thank you 💯